CommandConsole.cpp

```diff
@@ -290,6 +294,24 @@ bool CommandConsole::RunCommand(CommandGui *commandGui, const std::string& cmd)
     return false;
 }
 
+void CommandConsole::InputData(CommandGui *commandGui, int key)
+{
+    auto& inputBox = commandGui->inputBox;
+    char inputKey = key;
+
+    if (inputKey >= 'a' && inputKey <= 'z')
+    {
+        inputKey -= 32;
+    }
+    else if (inputKey >= 'A' && inputKey <= 'Z')
+    {
+        inputKey += 32;
+    }
```

Comment:
Well I'm not fixing it, I'm replicating it since everyone is used to having the FTL command console capitalise by default and all command require capitalized input (except crew name I believe, and lua command).
What we can do though is add an option enabled by default for that behaviour.